requirements.txt

```diff
@@ -100,8 +95,6 @@ pytest-metadata==3.0.0
     # via pytest-html
 pytest-timeout==2.1.0
     # via -r requirements-build.in
-pywin32-ctypes==0.2.2
```

Comment:
Maybe this should be changed as `pywin32-ctypes==0.2.2 ;  platform_system == 'Windows'`, instead of being removed. We might also want to keep colorama in case it comes from a windows requirements.txt update, although it doesn't appear to be windows specific (safe to keep as is).

Reply:
Python is annoying in that even if you do that in the `.in` files, when you generate on "not Windows" it still won't be included.  In addition, the transitive deps will also be removed.  I get the latter since it is hard to follow deps for something you aren't installing (though a more robust build system would have ways to solve that), but the fact that the root project is also ignored is.. a real PITA.  Overall I think the only strategy is to examine any removals (such as these) and manually remove them unless you know they are due to other changes and aren't platform specific.  Otherwise the only way to do it is to perform requirements generation on all possible platforms and merge the result (not something that is practical).

Reply:
I'll add those two back in with associated comments that say something about "manually" added.  Not sure that will really fix things since we regenerate the requirements.txt file when we determine it's necessary.  But perhaps that's all that's possible in this environment.

Reply:
@jcguion & @sfitts ... is this a reasonable solution?  If so, I'll assume it's OK, merge, and publish to pypi

Reply:
LGTM